better-font-awesome.php

```diff
@@ -487,6 +487,15 @@ public function admin_enqueue_scripts( $hook ) {
 	 * @since  1.0.10
 	 */
 	public function save_options() {
+		if ( false == check_ajax_referer( self::SLUG . '-options', 'bfa_nonce', false ) ) {
+				wp_die(
+					'<div class="error"><p>'
+					. __( 'Settings were not saved due to a missing nonce. Refresh the page and try again.', 'better-font-awesome' )
```

Comment:
@MickeyKay I implemented your text changes but additionally asked the user to refresh the page (because we are not doing it anymore).